stylua-vscode/src/util.ts

```diff
@@ -143,10 +166,11 @@ export const ensureStyluaExists = async (
     }
 
     try {
-      const version = (await executeStylua(path, ["--version"]))?.trim();
-      const release = await getLatestRelease();
+      const currentVersion = (await executeStylua(path, ["--version"]))?.trim();
+      const desiredVersion = getDesiredVersion();
+      const release = await getRelease(desiredVersion);
       if (
-        version !==
+        currentVersion !==
```

Comment:
I think we still need a way to let the user know that there is a new *latest* release to StyLua, even if they aren't set to latest. It can keep them up to date.
But this will keep showing every time the extension becomes activated, so there should be an option along the lines of "dont remind me for this version", where we save a config to not show the prompt again if the user ignored it.

Reply:
This is a good concept, but quickly became out of scope with several pitfalls. I've created #216 to track this.